apps/src/templates/VersionHistory.jsx

```diff
@@ -132,35 +133,47 @@ export default class VersionHistory extends React.Component {
   };
 
   render() {
+    let title;
     let body;
     if (this.state.showSpinner) {
+      title = i18n.versionHistory_header();
       body = (
         <div style={{margin: '1em 0', textAlign: 'center'}}>
           <i className="fa fa-spinner fa-spin" style={{fontSize: '32px'}} />
         </div>
       );
     } else if (this.state.confirmingClearPuzzle) {
+      title = i18n.versionHistory_clearProgress_header();
       body = (
         <div>
           <p>{i18n.versionHistory_clearProgress_prompt()}</p>
+          {this.props.isProjectTemplateLevel && (
+            <p id="template-level-warning">
+              {i18n.versionHistory_clearProgress_templateLevelWarning()}
+            </p>
+          )}
           <button
             type="button"
-            id="confirm-button"
-            style={{float: 'right'}}
+            className="btn-danger"
+            id="start-over-button"
```

Comment:
I needed to change the id of this button to avoid picking up styling associated with `.confirm-button`.

Reply:
Just curious, where is it getting the `confirm-button` class?

Reply:
Here I think: https://github.com/code-dot-org/code-dot-org/blob/94af9bdcf767d9bbd60379e64bc41255bbaae494/apps/style/common.scss#L1279

Unfortunately, this change is causing some UI tests to fail so I'm trying to see if it's easier for me to fix the tests or to revert this change and override the style.

Reply:
So an option here, instead of adding the id, you could add to the inline styles since inline styles will override id styles (https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
Sorry, forgot to close the loop here.  I ended up using a class-based style as Tim suggested.  I do think that inline styles is ultimately the way we want to go here, but wanted to make the minimal change that follows as closely to current patterns.